src/libraries/sendtohelix-wasm.targets

```diff
@@ -70,6 +81,15 @@
     <_HelixLocalNodePath Condition="'$(NeedsEMSDK)' == 'true' and '$(WindowsShell)' == 'true'">%HELIX_CORRELATION_PAYLOAD%\build\emsdk\node</_HelixLocalNodePath>
   </PropertyGroup>
 
+  <ItemGroup>
+    <HelixDependenciesToStage
```

Comment:
why there is no WASI SDK here ? We don't need any ?
why there is no wasmtime here ?

Reply:
Scratch it, I see it above now.